configs/ttalps_analysis/ttalps_histogrammer_default_config.py

```diff
@@ -28,14 +29,14 @@
   ("Event"              , "nMuon"               , 50    , 0     , 50    , ""  ),
   ("Muon"               , "pt"                  , 2000  , 0     , 1000  , ""  ),
   ("Muon"               , "eta"                 , 100   , -2.5  , 2.5   , ""  ),
-  ("Muon"               , "dxy"                 , 1600  , -20   , 20    , ""  ),
-  ("Muon"               , "dz"                  , 1600  , -20   , 20    , ""  ),
+  ("Muon"               , "dxy"                 , 1600  , -400  , 400   , ""  ),
+  ("Muon"               , "dz"                  , 1600  , -400  , 400   , ""  ),
```

Comment:
Shouldn't we extend it to the limit of CMS, like 10 meters or something like that? And we should probably adjust number of bins, so we can plot it with finer binning close to the PV if needed.